src/www/system_advanced_admin.php

```diff
@@ -636,6 +714,16 @@ function reloadWaitOld () {
                   </div>
                 </td>
               </tr>
+              <tr class="webgui_managementaccess_choices webgui_managementaccess_ip">
+                <td><a id="help_for_webguiip" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?= gettext("Management IP") ?></td>
+                <td>
+                  <input name="webguiip" type="text" value="<?= $pconfig['webguiip'] ?>"/>
+                  <?=gettext("Only accept connections from the specified IP. Use with care."); ?>
+                  <div class="hidden" data-for="help_for_webguiip">
+                    <?= gettext('The WebGUI only listens on the selected IP address. You may need to manually add required firewall rules first. Use with care.') ?>
```

Comment:
```suggestion
                    <?= gettext('The Web GUI only listens on the selected IP address. You may need to manually add required firewall rules first. Use with care.') ?>
```

Reply:
Actually the wording is not consistent in OPNsense:

opnsense/core> grep -Ri 'WebGUI'|wc -l
237
opnsense/core> grep -Ri 'Web GUI'|wc -l
30

"WebGUI" is used more often, so I'd prefer this one.